src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs

```diff
@@ -562,6 +580,21 @@ public override void ExecuteCmdlet()
             }
         }
 
+        private void Validate()
+        {
+            if (MyInvocation.BoundParameters.ContainsKey(nameof(UseDeviceAuthentication))
+                && IsWriteInformationIgnored())
+            {
+                throw new ActionPreferenceStopException(Resources.DoNotIgnoreInformationIfUserDeviceAuth);
+            }
+        }
+
+        private bool IsWriteInformationIgnored()
```

Comment:
```suggestion
        static private bool IsWriteInformationIgnored()
```

Reply:
can not be static as it's bounded with cmdlet's MyInvocation.BoundParameters and SessionState